langchain4j-worker-ai/src/main/java/dev/langchain4j/model/workerai/WorkerAiModelName.java

```diff
@@ -0,0 +1,73 @@
+package dev.langchain4j.model.workerai;
+
+/**
+ * Group model name in a class to ease the usage.
+ * It has been preferred as an Enum for flexibility.
+ */
+@SuppressWarnings("unused")
+public class WorkerAiModelName {
+
+    /**
+     * Max token for text generation
+     */
+    public static final int MAX_TOKENS =  512;
+
+    // ---------------------------------------------------------------------
+    // Text Generation
+    // https://developers.cloudflare.com/workers-ai/models/text-generation/
+    // ---------------------------------------------------------------------
+    /**
+     * Model names
+     */
+    public static final String LLAMA2_7B_FULL = "@cf/meta/llama-2-7b-chat-fp16";
+    /**
+     * Model names
+     */
+    public static final String LLAMA2_7B_QUANTIZED = "@cf/meta/llama-2-7b-chat-int8";
+    /**
+     * Model names
+     */
+    public static final String LLAMA2_7B_AWQ = "@hf/thebloke/codellama-7b-instruct-awq";
+    /**
+     * Model names
+     */
+    public static final String MISTRAL_7B_INSTRUCT = "@cf/mistral/mistral-7b-instruct-v0.1";
+
+    // ---------------------------------------------------------------------
+    // Text Embeddings
+    // https://developers.cloudflare.com/workers-ai/models/text-embeddings/
+    // ---------------------------------------------------------------------
+
+    /**
+     * Model names
+     * maxToken: 512, dimension 384
+     */
+    public static final String BAAI_EMBEDDING_SMALL = "@cf/baai/bge-small-en-v1.5";
```

Comment:
minor: I would include versions of the models in the constant names as there is a high chance that another version will come soon